packages/jellyfish-api-core/src/category/net.ts

```diff
@@ -29,6 +38,38 @@ export class Net {
   }
 }
 
+export interface PeerInfo {
+  id: number
+  addr: string
+  addrbind: string
+  addrlocal: string
```

Comment:
I am wondering if these should be set as potentially undefined when looking at the [net.cpp](https://github.com/DeFiCh/ain/blob/master/src/rpc/net.cpp#L145-L148) implementation?